docs/src/main/sphinx/release/release-419.md

```diff
@@ -1,15 +1,29 @@
-# Release 419 (24 May 2023)
+# Release 419 (2 June 2023)
 
 ## General
 
 * Add the {func}`array_histogram` function to find the number of occurrences of
   the unique elements in an array. ({issue}`14725 `)
+* Improve planning performance for queries involving joins. ({issue}`17458`)
 * Fix query failure when the server JSON response exceeds the 5MB limit for
   string values. ({issue}`17557`)
 
+## Web UI
+
+* Allow uppercase or mixed case values for the `web-ui.authentication.type`
+  configuration property. ({issue}`17334`)
+
+## BigQuery connector
+
+* Add support for proxying BigQuery APIs via an HTTP(S) proxy. ({issue}`17508`)
+* Improve performance of retrieving metadata from BigQuery. ({issue}`16064`)
+
 ## Delta Lake connector
 
+* Support adding a new column with the `id` and `name` column mapping modes. ({issue}`17236`)
```

Comment:
Are those two mutually exclusive modes?

Maybe rephrase as "Support the id and name mapping modes when adding new columns".